packages/core/src/@types/queryprovider.ts

```diff
@@ -59,7 +59,7 @@ export interface IPoolDate {
 }
 
 /** The fee structure, denoted as an array of numerator and denominator. */
-export type TFee = [number, number];
```

Comment:
This won't have any effect on the v1 createPool, will it? Or will we switch to the new `mintPool` fn for v1 pools as well?

Reply:
Per our conversation, there will be no `mintPool` method for v1 contracts.